projects/backend/src/plugins/oauth.js

```diff
@@ -0,0 +1,84 @@
+'use strict'
+
+const oauthPlugin = require('@fastify/oauth2')
+
+module.exports = async function spotifyOauthPlugin (app, opts, next) {
```

Comment:
we choose one interface async or callback style

```suggestion
module.exports = async function spotifyOauthPlugin (app, opts) {
```